src/sound/LICENCE.md

```diff
@@ -0,0 +1,22 @@
+# Sound effect licences
```

Comment:
Where is 'deer' from?

Reply:
@anoadragon453, and I think he said it wasn't from a source we could license. I think we'll have to take it out unless we can find a better deer.

Reply:
This should still be noted in the list somewhere

Reply:
I think it was replaced by a free one in https://github.com/element-hq/element-call/pull/2708/commits/1062b982893e30152141f3263f6740ad327b9b2f